spec/ship_spec.rb

```diff
@@ -1,19 +1,49 @@
-require 'spec_helper'
+require_relative 'spec_helper'
 
 RSpec.describe Ship do
-  subject { described_class.new("Cruiser", 3) }
```

Comment:
great descriptive name change subject to ship - never removed the placeholder after getting the test to work.